make.js

```diff
@@ -607,7 +605,6 @@ function stripCommentHeaders(content, filename) {
   // Strip out all the vim/license headers.
   var notEndOfComment = '(?:[^*]|\\*(?!/))+';
   var reg = new RegExp(
-    '\n/\\* -\\*- Mode' + notEndOfComment + '\\*/\\s*' +
```

Comment:
add '\n' to the next line, we want to not affect very first comment

also can you fix the comment above -- it does not reflect the purpose of the code below